QgisModelBaker/libqgsprojectgen/dataobjects/layers.py

```diff
@@ -231,8 +231,11 @@ def isPureLinkTable(self, project):
         for relation in project.relations:
             if relation.referencing_layer != self:
                 continue
-            remaining_fields.remove(relation.referencing_field)
+            remaining_fields.discard(relation.referencing_field)
 
-        # Only Id field left?
-        return len(remaining_fields) <= 1
+        # Remove Id fields
+        remaining_fields.discard('t_id')
+        remaining_fields.discard('t_ili_tid')
```

Comment:
In geopackage this value is "T_Ili_Tid" but you can get this value from the current db_connector.

Reply:
ok thanks